pandas/tests/io/test_excel.py

```diff
@@ -706,7 +738,34 @@ def test_read_from_http_url(self, ext):
                'pandas/tests/io/data/test1' + ext)
         url_table = read_excel(url)
         local_table = self.get_exceldf('test1', ext)
-        tm.assert_frame_equal(url_table, local_table)
+
+        try:
```

Comment:
You've mentioned already but obviously don't want this try...except here

Reply:
Any assistance in figuring this out would be welcome

Reply:
Can you remove the except on the next iteration? Would help to see the issue show up in the CI logs

Reply:
move this out to a spearate test

Reply:
if its failing then just xfail it for now don't try/except